profiles/fscloud/README.md

```diff
@@ -1,8 +1,72 @@
+<!-- BEGIN MODULE HOOK -->
 # Financial Services Cloud Profile
 
 This is a profile for MongoDB that meets FS Cloud requirements.
 It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals with the following exception:
 - 3000205: Check whether Databases for MongoDB network access is restricted to a specific IP range.
-    - This is ignored because the CBR locks this down and CRA does not check this
+  - This is ignored because the CBR locks this down and CRA does not check this
```

Comment:
This is referencing an SCC v1 goal. But we have moved to SCC v2, so it should have listed here the rule IDs for any failing rule. So what you have in `cra-tf-validate-ignore-rules.json` should be listed here